javamelody-core/src/main/java/net/bull/javamelody/CacheInformations.java

```diff
@@ -177,7 +177,7 @@ private static boolean isEhcacheAvailable() {
 	private static boolean isEhcache27() {
 		try {
 			// ce Class.forName est nécessaire sur le serveur de collecte
-			Class.forName("net.sf.ehcache.statistics.StatisticsGateway");
+			Class.forName("net.sf.ehcache.config.TerracottaConfiguration");

```

Comment:
This change is not right, because the net.sf.ehcache.config.TerracottaConfiguration class already exists in ehcache v2.6.11.
So it is not a good criteria and it will be needed to find another way to say if ehcache v2.7 is present, even if the class net.sf.ehcache.statistics.StatisticsGateway is not found when using Liferay.